src/udev/udev-builtin-net_id.c

```diff
@@ -164,11 +166,16 @@ static int dev_pci_onboard(struct udev_device *dev, struct netnames *names) {
         if (attr)
                 dev_port = strtol(attr, NULL, 10);
 
+        /* kernel provided front panel port name for multiple port PCI device */
+        port_name = udev_device_get_sysattr_value(dev, "phys_port_name");
+
         s = names->pci_onboard;
         l = sizeof(names->pci_onboard);
         l = strpcpyf(&s, l, "o%d", idx);
         if (dev_port > 0)
                 l = strpcpyf(&s, l, "d%d", dev_port);
+        if (port_name)
+                l = strpcpyf(&s, l, "n%s", port_name);
```

Comment:
There is no in-tree driver implementing both. And it does not really make sense for driver to do that. dev_port will slowly die, phys_port_name is to be used for new drivers.